app/src/main/res/values/help.xml

```diff
@@ -5,56 +5,49 @@
 &lt;i&gt;https://github.com/sky-map-team/stardroid&lt;/i&gt;&lt;br/&gt;&lt;br/&gt;
 
 &lt;b&gt;Introduction and Quick Start&lt;/b&gt;&lt;br/&gt;
-Sky Map turns your Android-powered mobile phone into a window on the
-night sky.  When you point your phone up you will see a map of the brightest
-stars, constellations, and planets in that part of the sky.&lt;br/&gt;Next time you see a
-bright  star and want to know what it\'s called, use Sky Map!&lt;br/&gt;
+Sky Map is your catalog of the night sky! Point your Android-powered device up, and you will see a map of the brightest
+stars, constellations, and planets in that part of the sky.&lt;br/&gt;Next time  what to identify a bright star or planet, use Sky Map!&lt;br/&gt;
 
 &lt;b&gt;Visible Layers&lt;/b&gt;&lt;br/&gt;
-
 Sky Map organizes the objects it shows you into layers.  Each layer can
 be toggled using the on-screen controls.  The layers are:&lt;br/&gt;
-      Stars&lt;br/&gt;
-      Constellations&lt;br/&gt;
-      Messier Objects&lt;br/&gt;
-      Planets&lt;br/&gt;
-      Meteor Showers&lt;br/&gt;
-      Right Ascension and Declination Grid&lt;br/&gt;
-      Horizon and cardinal points&lt;br/&gt;&lt;br/&gt;
-The brightest stars and most prominent constellations are labeled, as are the
-planets.  The planets are displayed as images that are larger than
-actual size.  The horizon is represented as a straight line.  Unless you live
-somewhere that is completely flat and free of buildings, it is likely that
-Sky Map will show you objects above the horizon line that you are actually
-unable to see.&lt;br/&gt;&lt;br/&gt;
+      <span>&#8226;</span>Stars&lt;br/&gt;
+      <span>&#8226;</span>Constellations&lt;br/&gt;
+      <span>&#8226;</span>Messier Objects&lt;br/&gt;
+      <span>&#8226;</span>Planets&lt;br/&gt;
+      <span>&#8226;</span>Meteor Showers&lt;br/&gt;
+      <span>&#8226;</span>Right Ascension and Declination Grid&lt;br/&gt;
+      <span>&#8226;</span> Horizon and cardinal points&lt;br/&gt;&lt;br/&gt;
+      
+The brightest stars, constellations, and planets are labeled. 
+Planets display as images that are larger than will appear in the sky.
+Sky Map may show objects that are above the horizon line (represented as a straight line)
+that you may not be able to use, unless you live in an area that is  flat and without buildings.&lt;br/&gt;&lt;br/&gt;
 
 &lt;b&gt;On-screen controls&lt;/b&gt;&lt;br/&gt;
-Tap on the screen to show the on-screen controls.  On one side of the screen are
+Tap on the screen to show the on-screen controls. On the side of the screen are
 the toggles for the layers described above.  They are orange when the layers are
 enabled and white when disabled.&lt;br/&gt;
 
-Along the bottom of the screen are zoom controls, and a toggle to switch between
+The bottom of the screen displays zoom controls and a toggle to switch between
```

Comment:
Another out of date sentence. There are no zoom controls any more (we use the standard "pinch" gesture).  And the auto/manual toggle now lives on the right hand side of the screen.